src/saturn/saturn_projectfile.cpp

```diff
@@ -0,0 +1,450 @@
+    { "k_shade_x", { { &world_light_dir1, nullptr }, "Mario Shade X" } },
+    { "k_shade_y", { { &world_light_dir2, nullptr }, "Mario Shade Y" } },
+    { "k_shade_z", { { &world_light_dir3, nullptr }, "Mario Shade Z" } },
+    { "k_shade_t", { { &world_light_dir4, nullptr }, "Mario Shade Tex" } },
+    { "k_scale", { { &marioScaleSizeX, nullptr }, "Mario Scale" } },
+    { "k_scale_x", { { &marioScaleSizeX, nullptr }, "Mario Scale X" } },
+    { "k_scale_y", { { &marioScaleSizeY, nullptr }, "Mario Scale Y" } },
+    { "k_scale_z", { { &marioScaleSizeZ, nullptr }, "Mario Scale Z" } },
+    { "k_head_rot", { { nullptr, &enable_head_rotations }, "Head Rotations" } },
+    { "k_v_cap_emblem", { { nullptr, &show_vmario_emblem }, "M Cap Emblem" } },
+    { "k_angle", { { (float*)&gMarioState->faceAngle[1], nullptr }, "Mario Angle" } },
+    { "k_hud", { { nullptr, &configHUD }, "HUD" } },
+    { "k_fov", { { &camera_fov, nullptr }, "FOV" } },
+    { "k_focus", { { &camera_focus, nullptr }, "Follow" } },
+    { "k_c_camera_pos0", { { &freezecamPos[0], nullptr }, "Camera Pos X" } },
+    { "k_c_camera_pos1", { { &freezecamPos[1], nullptr }, "Camera Pos Y" } },
+    { "k_c_camera_pos2", { { &freezecamPos[2], nullptr }, "Camera Pos Z" } },
+    { "k_c_camera_yaw", { { &freezecamYaw, nullptr }, "Camera Yaw" } },
+    { "k_c_camera_pitch", { { &freezecamPitch, nullptr }, "Camera Pitch" } },
+    { "k_c_camera_roll", { { &freezecamRoll, nullptr }, "Camera Roll" } }
+};
+
+uchar get_int8(char* data, int*offset) {
+    union {
+        char x;
+        uchar y;
+    } u;
+    u.x = data[*offset];
+    *offset += 1;
+    return u.y;
+}
+uint get_int32(char* data, int* offset) {
+    uint value = (get_int8(data, offset) << 24) | (get_int8(data, offset) << 16) | (get_int8(data, offset) << 8) | get_int8(data, offset);
+    return value;
+}
+ushort get_int16(char* data, int* offset) {
+    ushort value = (get_int8(data, offset) << 8) | get_int8(data, offset);
+    return value;
+}
+void get_string(char* data, int* offset, char* dest, int length) {
+    int i = 0;
+    do {
+        dest[i] = data[*offset + i];
+        i++;
+    }
+    while (data[*offset + i] != 0 && i < length);
+    dest[i] = 0;
+    *offset += i;
+}
+float get_floating_point(char* data, int* offset) {
+    union {
+        uint x;
+        float y;
+    } u;
+    u.x = get_int32(data, offset);
+    return u.y;
+}
+bool get_bool(char* data, int* offset) {
+    bool value = data[*offset] != 0;
+    *offset += 1;
+    return value;
+}
+void identifier(char* data, int* offset, char* identifier) {
+    get_string(data, offset, identifier, SATURN_PROJECT_IDENTIFIER_LENGTH);
+}
+
+void put_int32(char* data, int* offset, uint value) {
+    data[*offset] = (value >> 24) & 0xFF;
+    data[*offset + 1] = (value >> 16) & 0xFF;
+    data[*offset + 2] = (value >> 8) & 0xFF;
+    data[*offset + 3] = value & 0xFF;
+    *offset += 4;
+}
+void put_int16(char* data, int* offset, ushort value) {
```

Comment:
Same issue as lines [100](https://github.com/Llennpie/Saturn/blob/f987112d8bf0845917a5dc8266ab123e64f72400/src/saturn/saturn_projectfile.cpp#L100) and [101](https://github.com/Llennpie/Saturn/blob/f987112d8bf0845917a5dc8266ab123e64f72400/src/saturn/saturn_projectfile.cpp#L101).